### DISCUSSION TITLE
Weird issue... bug?

### DISCUSSION THREAD

**Discussion by jburf** [smplayer-dev/smplayer#485] [General]
Been using smplayer for centuries... thanks for all the efforts!
After updating to v22.2 from v21.10 (64-bit), I noticed that the video equalizer didn't do anything, when I moved the sliders. If I unchecked "software equalizer" in the equalizer itself, then the sliders would work, but if I played other videos, the remembered equalizer settings were way off. So, I reverted to 21.10, waiting for another update.
Today, I updated from 21.10 to v22.7, and encountered the same problem.
Was there some change in 22.2 from 21.10 that would cause this issue?
Any ideas? Any help appreciated!
(The computer I'm using is still on Win7, if that could be part of the issue)

**Comment by smplayer-dev** [comment]
In version 21.10 the default video output was direct3d, now the default video output is gpu. I guess if you select direct3d as video output you'll get the same result than before.

**Reply by jburf** [reply]
Thanks! That worked, however, while the video plays fine, when I close any video, I get an app crash. No app crash with v22.2, or 21.10, only 22.7...


> Problem Event Name:	APPCRASH
  Application Name:	smplayer.exe
  Application Version:	22.7.0.0
  Application Timestamp:	62cedaa1
  Fault Module Name:	smplayer.exe
  Fault Module Version:	22.7.0.0
  Fault Module Timestamp:	62cedaa1
  Exception Code:	c0000005
  Exception Offset:	0000000000236b53
  OS Version:	6.1.7601.2.1.0.256.48
  Locale ID:	1033
  Additional Information 1:	7821
  Additional Information 2:	7821d8196dc12b6c3b812baa6c34150c
  Additional Information 3:	6bf1
  Additional Information 4:	6bf109b3c1d9013c1383a71ddefed2cd

Any ideas? Thanks again!

**Comment by smplayer-dev** [comment]
Could you check if the crash happens as well with this package?
https://github.com/smplayer-dev/smplayer/releases/download/v22.7.0/smplayer-22.7.0-x64-qt5.6.exe

**Reply by jburf** [reply]
That one works, no crashing... thanks! 
What actually is the difference... a newer or older version of qt?

**Reply by smplayer-dev** [reply]
It uses an old version of Qt.

**Reply by jburf** [reply]
Thanks... appreciated! One last question, for the future, do you always create a release with the older Qt?
If not, is there anything on my end I can do, to make the computer compatible with newer versions of Qt
(outside of a newer computer/OS)... Thanks again for your help, & all your efforts on SMP!!

**Reply by smplayer-dev** [reply]
I'll try to fix that crash.
